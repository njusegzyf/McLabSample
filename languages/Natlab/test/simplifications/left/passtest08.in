
a = 3;
b(a) = foo();
