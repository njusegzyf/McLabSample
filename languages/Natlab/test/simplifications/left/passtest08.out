
a = 3;
t0 = foo();
b(a) = t0;